### PR TITLE
feat(content): add actionable language page

### PR DESCRIPTION
**Pull request by Francois-Esquire**
## Description

This PR adds a new content section "Actionable language".

## Detail

Adding the new section "Actionable language" and removing some of the words from the "Word list" page for clarity.
The actionable language has been put under a new sub nav category "Guides".

## Checklist

- [ ] :ok_hand: ~~website updates are Garden Designer approved (add the designer as a reviewer)~~
- [x] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [x] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- [ ] :wheelchair: ~~analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver~~
- [ ] :memo: ~~tested in Chrome, Firefox, Safari, Edge, and IE11~~

### CODE REVIEW

File: src/pages/content/actionable-language.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+### Skip
+
+Use “Skip” to allow the user to postpone an action or decision.
+It’s a way for the user to avoid interruption and proceed with their current task.
+
+## To receive confirmation
+
+### Done
+
+Use “Done” to confirm the completion of a multi-step process.
```
Comment (jzempel) [inline src/pages/content/actionable-language.mdx:54]: Does this conflict with https://garden.zendesk.com/content/word-list#done?

File: src/pages/content/actionable-language.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+### Cancel
+
+Use “Cancel” when the user makes edits but wants to discard them. It dismisses
+the page the user was editing and returns the user to the entry point they came from.
+
+- **Don’t use:** Discard changes
+
+### Leave without saving
+
+“Leave without saving” is similar to “Cancel'', with one key difference: it
```
Comment (jzempel) [inline src/pages/content/actionable-language.mdx:95]: Invalid curly end quote after "Cancel". Note that straight quotes may be used, and Gatsby will transform to proper curly form during the build.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: src/pages/content/actionable-language.mdx (hunk 2 above)
Comment (Francois-Esquire) [reply src/pages/content/actionable-language.mdx:95]: Good catch, will update this.

File: src/pages/content/actionable-language.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+Use “Skip” to allow the user to postpone an action or decision.
+It’s a way for the user to avoid interruption and proceed with their current task.
+
+## To receive confirmation
+
+### Done
+
+Use “Done” to confirm the completion of a multi-step process.
+
+- **Don’t use:** OK, Got it
```
[the remark this answers is not in the document]
Comment (diana-oum) [reply src/pages/content/actionable-language.mdx:56]: Thanks @jzempel !

@Francois-Esquire - I will bring this to Chelsea and will let you know how to proceed.

File: src/pages/content/actionable-language.mdx (hunk 1 above)
Comment (diana-oum) [reply src/pages/content/actionable-language.mdx:54]: Yes it does, thanks for catching @jzempel 
@Francois-Esquire - Same thing here: I'll bring to Chelsea and let you know on how to resolve this!

Comment (diana-oum) [reply src/pages/content/actionable-language.mdx:54]: @Francois-Esquire I'm back! And with confirmation from Chelsea on my changes.

I forgot to mention to delete Done from Garden word list when we were jamming together. Sorry! Could you please remove it 🙏

File: src/pages/content/actionable-language.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (diana-oum) [reply src/pages/content/actionable-language.mdx:56]: @Francois-Esquire For this issue, add this new "OK" entry below "Done". Please also include "Close" in Done's Don't use bullet point. See reference:

image

File: src/pages/content/actionable-language.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+## To learn
+
+### Find out more
+
+Use “Find out more” to launch a modal that introduces a new feature.
+
+### Learn about
+
+Launch a link to the Zendesk help center.
```
Comment (diana-oum) [inline src/pages/content/actionable-language.mdx:114]: @Francois-Esquire After reviewing the documentation again, I've added a few more lines for "Learn about"'s definition:

Use “Learn about” to include a link to the Zendesk help center. Have the link be a few words about what the user will learn. Avoid the standalone “Learn more” and help center article titles that can’t be localized.

See [reference](https://docs.google.com/document/d/19KhaviU-9Jt_LZqV2k7nm1lx9KYi1XEemQhMUPSFovA/edit#heading=h.dohlqu9z657v):
image

**Review by diana-oum — CHANGES_REQUESTED** [review]

File: src/pages/content/actionable-language.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (diana-oum) [reply src/pages/content/actionable-language.mdx:56]: @Francois-Esquire Please also remove "OK" from Garden word list!

**Review by diana-oum — APPROVED** [review]
:shipit:

**Review by jzempel — APPROVED** [review]

**Pull request merged**